guides/common/modules/proc_verifying-pcp-configuration.adoc

```diff
@@ -1,15 +1,15 @@
 [id='verifying-pcp-configuration_{context}']
 = Verifying PCP Configuration
 
-To verify PCP is configured correctly, and services are active, run the following command:
+To verify PCP is configured correctly and services are active, run the following command:
 
 ----
 # pcp
 ----
 
 This outputs a summary of the active PCP configuration.
+Example output of the `pcp` command:
```

Comment:
```suggestion

Example output of the `pcp` command:
```

Reply:
@adamlazik1 I did it a bit differently, let me know if it works.

Reply:
That works, thanks!